### ISSUE TITLE
Failed to compile. TypeError: Cannot set properties of undefined (setting 'kind')

### ISSUE THREAD

**Issue by trev-gulls** [floating-ui/floating-ui#2067]
I'm getting a TypeError from a nested dependency of `@aws-amplify/ui-react` using `@radix-ui/react-dropdown-menu` ... `@floating-ui/dom` using `@floating-ui/core` introduced by radix-ui which was introduced by . The error prevents successful react-scripts build. I've not been able to determine the exact ui component attempting to set an undefined property. Build output and dependency chain shown below:
```
> react-scripts build

Creating an optimized production build...
Failed to compile.

TypeError: /.../project/node_modules/@floating-ui/dom/node_modules/@floating-ui/core/dist/floating-ui.core.browser.min.mjs: Cannot set properties of undefined (setting 'kind')
    at transformFile.next (<anonymous>)
    at run.next (<anonymous>)
    at transform.next (<anonymous>)
```

```
aws-amplify-app@0.1.0 /.../project
└─┬ @aws-amplify/ui-react@4.3.1
  └─┬ @radix-ui/react-dropdown-menu@1.0.0
    └─┬ @radix-ui/react-menu@1.0.0
      └─┬ @radix-ui/react-popper@1.0.0
        └─┬ @floating-ui/react-dom@0.7.2
          └─┬ @floating-ui/dom@0.5.4
            └── @floating-ui/core@0.7.3

```

**Comment by atomiks** [comment]
This was a bug in `@babel/plugin-transform-block-scoping@7.20.7` and `7.20.8`, a fix was released in `7.20.9`

**Comment by trev-gulls** [comment]
> This was a bug in `@babel/plugin-transform-block-scoping@7.20.7` and `7.20.8`, a fix was released in `7.20.9`

Thank you!
